ZenPacks/zenoss/ZenPackLib/lib/functions.py

```diff
@@ -302,22 +318,38 @@ def relationships_from_yuml(yuml):
 
 
 def getZenossKeywords(klasses):
-    kwset = set()
+    kwset = set(['_properties', '_relations'])
```

Comment:
Since this PR is for debug logging, can you pull this out into a separate PR since it's dealing with the keywords issue?